addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -6555,274 +6546,149 @@ stock GetBossIndex(client)
 
 stock ParseFormula(client, const String:key[], const String:defaultFormula[], defaultValue)
 {
-	decl String:formula[1024], String:buffer[2];
-	new String:strValue[1024], Float:sum[128], calcOperators:_operator[128];
-	new bool:closeBracket, bool:usePlayers, bool:foundOperator, bracket;
+	decl String:formula[1024], String:bossName[1024];
 	KvRewind(BossKV[Special[client]]);
+	KvGetString(BossKV[Special[client]], "name", bossName, sizeof(bossName), "");
 	KvGetString(BossKV[Special[client]], key, formula, sizeof(formula), defaultFormula);
 	ReplaceString(formula, sizeof(formula), " ", ""); //Get rid of spaces
-	new length=strlen(formula);
-	for(new i; i<=length; i++)
+	new result=RoundFloat(IterateFormula(formula, key, bossName));
+	Debug("result = %i", result);
+	if(result<=0)
 	{
-		strcopy(buffer, sizeof(buffer), formula[i]);
-		Debug("buffer[0] contains %c", buffer[0]);
-		switch (buffer[0])
+		Debug("MALFORMATION!");
+		LogError("[FF2] %s has a malformed %s formula, using default!", bossName, key);
+		return defaultValue;
+	}
+	if(bMedieval)
+	{
+		return RoundFloat(result/3.6); //TODO: Make this configurable
+	}
+	return result;
+}
+
+enum Operators {
+	Operator_None = -1,
+	Operator_Add,
+	Operator_Subtract,
+	Operator_Multiply,
+	Operator_Divide,
+	Operator_Exponent,
+};
+
+stock Operate(Handle:sum, bracket, Float:v2, Handle:_operator)
```

Comment:
sum here can be called sumArray and then the actual sum can just be called sum instead of oSum.
What's v2 for?